### PR TITLE
Use default service names in examples.

### PR DESCRIPTION
**Pull request by nishantmonu51**
Use default service names in druid examples. 
Make them consistent with service names in druid code. 
fixes https://github.com/druid-io/druid/issues/2046

### CODE REVIEW

**Comment by himanshug** [comment]
:+1:

**Comment by fjy** [comment]
@nishantmonu51 if you change things here, make sure tranquility still works by default

File: examples/config/_common/common.runtime.properties [hunk 1]
```diff
@@ -43,7 +43,10 @@ druid.cache.type=local
 druid.cache.sizeInBytes=10000000
 
 # Indexing service discovery
-druid.selectors.indexing.serviceName=overlord
+#druid.selectors.indexing.serviceName=druid:overlord
+
+# Coordinator service discovery
+#druid.selectors.indexing.serviceName=druid:coordinator
```
Comment (gianm) [inline examples/config/_common/common.runtime.properties]: this line is not right

**Comment by gianm** [comment]
Oops, I did #2052 before seeing this PR. I am biased toward that one though. It updates a few more things and also fixes the router docs.

**Comment by nishantmonu51** [comment]
closing in favor of #2052

**Pull request closed without merging**